manifests/kustomize/third-party/metacontroller/base/stateful-set.yaml

```diff
@@ -18,26 +18,28 @@ spec:
         sidecar.istio.io/inject: "false"
     spec:
       containers:
-      - command:
-        - /usr/bin/metacontroller
-        - --logtostderr
-        - -v=4
-        - --discovery-interval=20s
-        image: metacontroller/metacontroller:v0.3.0
-        imagePullPolicy: Always
-        name: metacontroller
-        ports:
-        - containerPort: 2345
```

Comment:
Why do we omit the port here?

Reply:
The question should be rather: Why was the port mentioned before? It is not in upstream https://github.com/metacontroller/metacontroller/blob/master/manifests/production/metacontroller.yaml and i do not see why the port belongs there. I think this is a relict of the old 0.3 version

Reply:
Sounds good to me.